.travis.yml

```diff
@@ -4,15 +4,21 @@
 language: generic
 
 os: osx
-osx_image: xcode6.4
+osx_image: beta-xcode6.1
 
 env:
   matrix:
     
+    - CONDA_NPY=110  CONDA_PY=27
     - CONDA_NPY=111  CONDA_PY=27
     - CONDA_NPY=112  CONDA_PY=27
+    - CONDA_NPY=110  CONDA_PY=34
+    - CONDA_NPY=111  CONDA_PY=34
+    - CONDA_NPY=112  CONDA_PY=34
```

Comment:
conda-forge is no longer support Python 3.4 and Numpy 1.10 did you add those manually? If so we should not add anything manually to the CIs configuration files.